### PR TITLE
Enhancement/issue 57 migrate rule page

### PR DESCRIPTION
**Pull request by elizabethh777**
Rule Definition page contains  4 parts (steps) : DID, RSE, Options, Summary/Submit rule definition.

What is left for this page:
Finish the REST calls for the 2nd (`RSE`) step, client functionality and model are already there. Do the REST call for 4th step (`Submit`).

### CODE REVIEW

**Comment by maany** [comment]
Could you please fix the CI checks ( Building and Linting)

File: src/rucio_client/accountlimits.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export function get_account_limits(account: string, rucioToken: string) {
+    const url = '/accounts/' + account + '/limits'
+    return streamData(
+        url,
+        {},
+        {
+            'X-Rucio-Auth-Token': rucioToken,
+        },
+    )
```
Comment (nimishbongale) [inline src/rucio_client/accountlimits.ts:11]: Any particular reason why we decided to create another client module parallel to the one that already exists?

Comment (maany) [reply src/rucio_client/accountlimits.ts:11]: Hey @nimishbongale, this PR is still a WIP, however, @elizabethh777 has moved away from the project. So we use this as a foundation for the Rules pages and finish the pending REST connections and processing of data. ;) 

In this case, this part should be moved to the existing client module. @elizabethh777 wrote this while we were still waiting for the client module ;)

**Review by nimishbongale — APPROVED** [review]
Will take up subsequent refactoring as a part of a separate PR

**Pull request merged**